test/vintage_net_wizard/web/api_test.exs

```diff
@@ -308,6 +301,11 @@ defmodule VintageNetWizard.Web.ApiTest do
 
     {conn, body} = run_request(:get, "/complete")
 
+    # Starts a task to kill the server after delivery
+    assert Task.Supervisor.children(VintageNetWizard.TaskSupervisor)
```

Comment:
`assert length(Task.Supervisor.children(VintageNetWizard.TaskSupervisor)) == 1`

Reply:
I can do that. Its just personal preference here for styling and I tend to learn towards pipes.

Reply:
I do wonder whether `ex_unit`'s assertion failure pretty printer can untangle the pipe to `Kernel.==/2`. I assume that it will just say that the test failed rather than saying what was on the left and right sides of the test.

Reply:
Looks like its not as useful:
```  1) test completes configuration (VintageNetWizard.Web.ApiTest)
     test/vintage_net_wizard/web/api_test.exs:294
     Expected truthy, got false
     code: assert Task.Supervisor.children(VintageNetWizard.TaskSupervisor) |> length() |> Kernel.==(2)
     stacktrace:
       test/vintage_net_wizard/web/api_test.exs:305: (test)
```
Making the change

Reply:
Fixed